index.js

```diff
@@ -8,7 +8,10 @@ const exclusions = ['--help'];
 
 module.exports = function (cb) {
   try {
-    cb(null, require(tmpfile));
+    var flags = require(tmpfile);
```

Comment:
This isn't necessary at all, unless I am missing something. `require` is evaluated before the callback is called. If it throws, the callback will never be called. 

Try this:
```js
function cb (err, result) {
  console.log('callback called', err, result);
}

try {
  cb(null, require('akjhasdk'));
} catch (e) {
  console.log('require failed.');
}
```